.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!---
+Pull Request Template
+- Please fill the sections below as described in the comments.
+- This information simplifies collaboration and helps future readers.
+- Reviewers should ask for this info to be filled when missing.
+-->
+
+<!---
+PR Metadata
+- Title: Provide a short summary of the PR changes.
+- Assignees: Assign yourself and other direct collaborators.
+- Type: Create the PR as a draft. Only set it to non-draft when it is ready to be reviewed.
+- Reviewers: Only assign reviewers when the PR is ready to be reviewed.
+- Labels: Only assign labels with high priority and important context. Default is label-less.
+-->
+
+### Check the Pull Request Type:
+<!--- Put an `x` in the box that applies. -->
+
+1. [ ] Implementation of new feature(s)
+2. [ ] Bug fixes or enhancements
```

Comment:
I think this section can be removed or be refactored. 

There are PRs that won't mark all those checks, making the PR list show incomplete checklist counts. For example, the [PR list for this repo](https://github.com/DeepX-inc/.github/pulls) shows that `7 of 10 tasks` are completed for this PR. This makes the checklist counter misleading for readers taking a look there.

We should use checklists that are intended to be completely marked. Such that the PR list shows `10 of 10 tasks` instead of `9 of 10 tasks`.